To: vim_dev@googlegroups.com
Subject: Patch 7.3.188
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.188
Problem:    More RISC OS files to remove.
Solution:   Remove them.  Update the file list.
Files:	    src/proto/gui_riscos.pro, src/proto/os_riscos.pro, Filelist


*** ../vim-7.3.187/src/proto/gui_riscos.pro	2010-08-15 21:57:28.000000000 +0200
--- src/proto/gui_riscos.pro	1970-01-01 01:00:00.000000000 +0100
***************
*** 1,66 ****
- /* Prototypes for gui_riscos.c
-  * Based on gui_x11_pro.h (10 March 2002 version)
-  */
- void gui_mch_prepare __ARGS((int *argc, char **argv));
- int gui_mch_init_check __ARGS((void));
- int gui_mch_init __ARGS((void));
- void gui_mch_uninit __ARGS((void));
- void gui_mch_new_colors __ARGS((void));
- int gui_mch_open __ARGS((void));
- void gui_init_tooltip_font __ARGS((void));
- void gui_init_menu_font __ARGS((void));void gui_mch_exit __ARGS((int rc));
- int gui_mch_get_winpos __ARGS((int *x, int *y));
- void gui_mch_set_winpos __ARGS((int x, int y));
- void gui_mch_set_shellsize __ARGS((int width, int height, int min_width, int min_height, int base_width, int base_height));
- void gui_mch_get_screen_dimensions __ARGS((int *screen_w, int *screen_h));
- int gui_mch_init_font __ARGS((char_u *font_name, int do_fontset));
- GuiFont gui_mch_get_font __ARGS((char_u *name, int giveErrorIfMissing));
- int gui_mch_adjust_charheight __ARGS((void));
- void gui_mch_set_font __ARGS((GuiFont font));
- void gui_mch_set_fontset __ARGS((GuiFontset fontset));
- void gui_mch_free_font __ARGS((GuiFont font));
- void gui_mch_free_fontset __ARGS((GuiFontset fontset));
- GuiFontset gui_mch_get_fontset __ARGS((char_u *name, int giveErrorIfMissing, int fixed_width));
- guicolor_T gui_mch_get_color __ARGS((char_u *reqname));
- void gui_mch_set_fg_color __ARGS((guicolor_T color));
- void gui_mch_set_bg_color __ARGS((guicolor_T color));
- void gui_mch_draw_string __ARGS((int row, int col, char_u *s, int len, int flags));
- int gui_mch_haskey __ARGS((char_u *name));
- void gui_mch_beep __ARGS((void));
- void gui_mch_flash __ARGS((int msec));
- void gui_mch_invert_rectangle __ARGS((int r, int c, int nr, int nc));
- void gui_mch_iconify __ARGS((void));
- void gui_mch_set_foreground __ARGS((void));
- void gui_mch_draw_hollow_cursor __ARGS((guicolor_T color));
- void gui_mch_draw_part_cursor __ARGS((int w, int h, guicolor_T color));
- void gui_mch_update __ARGS((void));
- int gui_mch_wait_for_chars __ARGS((long wtime));
- void gui_mch_flush __ARGS((void));
- void gui_mch_clear_block __ARGS((int row1, int col1, int row2, int col2));
- void gui_mch_clear_all __ARGS((void));
- void gui_mch_delete_lines __ARGS((int row, int num_lines));
- void gui_mch_insert_lines __ARGS((int row, int num_lines));
- void clip_mch_lose_selection __ARGS((VimClipboard *cbd));
- int clip_mch_own_selection __ARGS((VimClipboard *cbd));
- void clip_mch_request_selection __ARGS((VimClipboard *cbd));
- void clip_mch_set_selection __ARGS((VimClipboard *cbd));
- void gui_mch_menu_grey __ARGS((vimmenu_T *menu, int grey));
- void gui_mch_menu_hidden __ARGS((vimmenu_T *menu, int hidden));
- void gui_mch_draw_menubar __ARGS((void));
- void gui_mch_set_blinking __ARGS((long waittime, long on, long off));
- void gui_mch_stop_blink __ARGS((void));
- void gui_mch_start_blink __ARGS((void));
- void process_event __ARGS((int event, int *block));
- void gui_mch_show_popupmenu __ARGS((vimmenu_T *menu));
- long_u gui_mch_get_rgb __ARGS((guicolor_T pixel));
- void gui_mch_getmouse __ARGS((int *x, int *y));
- void gui_mch_setmouse __ARGS((int x, int y));
- void gui_mch_drawsign __ARGS((int row, int col, int typenr));
- void gui_mch_destroy_sign __ARGS((XImage *sign));
- void gui_mch_mousehide __ARGS((int hide));
- void mch_set_mouse_shape __ARGS((int shape));
- void gui_mch_menu_set_tip __ARGS((vimmenu_T *menu));
- 
- void ro_redraw_title __ARGS((int window));
- int ro_ok_to_quit __ARGS((void));
- /* vim: set ft=c : */
--- 0 ----
*** ../vim-7.3.187/src/proto/os_riscos.pro	2010-08-15 21:57:28.000000000 +0200
--- src/proto/os_riscos.pro	1970-01-01 01:00:00.000000000 +0100
***************
*** 1,49 ****
- /* os_riscos.c */
- void mch_write __ARGS((char_u *s, int len));
- int mch_inchar __ARGS((char_u *buf, int maxlen, long wtime, int tb_change_cnt));
- int mch_char_avail __ARGS((void));
- long_u mch_avail_mem __ARGS((int special));
- void mch_delay __ARGS((long msec, int ignoreinput));
- void mch_suspend __ARGS((void));
- void mch_init __ARGS((void));
- int mch_check_win __ARGS((int argc, char **argv));
- int mch_input_isatty __ARGS((void));
- int mch_can_restore_title __ARGS((void));
- int mch_can_restore_icon __ARGS((void));
- void mch_settitle __ARGS((char_u *title, char_u *icon));
- void mch_restore_title __ARGS((int which));
- int mch_get_user_name __ARGS((char_u *s, int len));
- void mch_get_host_name __ARGS((char_u *s, int len));
- long mch_get_pid __ARGS((void));
- int mch_dirname __ARGS((char_u *buf, int len));
- int mch_FullName __ARGS((char_u *fname, char_u *buf, int len, int force));
- int mch_isFullName __ARGS((char_u *fname));
- long mch_getperm __ARGS((char_u *name));
- int mch_setperm __ARGS((char_u *name, long perm));
- void mch_hide __ARGS((char_u *name));
- int mch_isdir __ARGS((char_u *name));
- int mch_can_exe __ARGS((char_u *name));
- int mch_nodetype __ARGS((char_u *name));
- void mch_early_init __ARGS((void));
- void mch_exit __ARGS((int r));
- void mch_settmode __ARGS((int tmode));
- void mch_setmouse __ARGS((int on));
- int mch_screenmode __ARGS((char_u *arg));
- int mch_get_shellsize __ARGS((void));
- void mch_set_shellsize __ARGS((void));
- void mch_new_shellsize __ARGS((void));
- int mch_call_shell __ARGS((char_u *cmd, int options));
- void mch_breakcheck __ARGS((void));
- int mch_expandpath __ARGS((garray_T *gap, char_u *path, int flags));
- int expand_section __ARGS((garray_T *gap, char_u *root, char_u *rest, int flags));
- int mch_expand_wildcards __ARGS((int num_pat, char_u **pat, int *num_file, char_u ***file, int flags));
- int mch_has_exp_wildcard __ARGS((char_u *p));
- int mch_has_wildcard __ARGS((char_u *p));
- int mch_remove __ARGS((char_u *file));
- char_u *mch_munge_fname __ARGS((char_u *fname));
- int ro_buflist_add __ARGS((char_u *old_name));
- int mch_chdir __ARGS((char_u *dir));
- void mch_read_filetype __ARGS((char_u *file));
- void mch_set_filetype __ARGS((char_u *file, char_u *type));
- int mch_check_filetype __ARGS((char_u *fname, char_u *type));
- /* vim: set ft=c : */
--- 0 ----
*** ../vim-7.3.187/Filelist	2011-03-22 18:10:34.000000000 +0100
--- Filelist	2011-05-10 17:19:21.000000000 +0200
***************
*** 428,435 ****
  		README_os390.txt \
  		src/Make_mint.mak \
  		src/Make_ro.mak \
- 		src/gui_riscos.c \
- 		src/gui_riscos.h \
  		src/if_sniff.c \
  		src/infplist.xml \
  		src/link.390 \
--- 428,433 ----
***************
*** 437,446 ****
  		src/os_beos.h \
  		src/os_beos.rsrc \
  		src/os_mint.h \
- 		src/os_riscos.c \
- 		src/os_riscos.h \
- 		src/proto/gui_riscos.pro \
- 		src/proto/os_riscos.pro \
  		src/os_vms_fix.com \
  		src/toolbar.phi \
  
--- 435,440 ----
*** ../vim-7.3.187/src/version.c	2011-05-10 16:41:13.000000000 +0200
--- src/version.c	2011-05-10 17:20:50.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     188,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
83. Batteries in the TV remote now last for months.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
